lib/concepts/feedback/operations/create.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class Feedback
+  class Create
+    class << self
+      def call(feedback_params:)
+        response = OperationResponse.new
+        response[:feedback] = build_feedback(feedback_params)
+        response[:feedback].save!
+        response
+      rescue StandardError => e
+        Sentry.capture_exception(e)
+        if response[:feedback].nil?
```

Comment:
I'm a bit confused by this, would it maybe be better to be checking is `.errors` is not empty first and doing the `join`, and then falling back to this? but maybe I'm misunderstanding something